app/assets/stylesheets/custom.css.scss

```diff
@@ -42,6 +42,15 @@ h1, h2, h3 {
     margin: 25px;
 }
 
+#clam-pic {
+    background: $dark-ocean-green;
+    border: 5px solid $dark-ocean-green;
+    margin-top: 40px;
+    margin-right: 150px;
+    padding:5px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space